src/zokrates.pest

```diff
@@ -9,12 +9,25 @@
 // Doc: associativity and precedence table for operators
 
 file = { SOI ~ NEWLINE* ~ import_directive* ~ NEWLINE* ~ function_definition* ~ EOI }
-import_directive = {"import" ~ "\"" ~ (!"\"" ~ ANY)* ~ "\"" ~ ("as" ~ identifier)? ~ NEWLINE+}
-function_definition = {"def" ~ identifier ~ "(" ~ parameter_list? ~ ")" ~ "->" ~ "(" ~ (type_name ~ ("," ~ type_name)*)? ~ ")" ~ ":" ~ NEWLINE* ~ statement* }
+import_directive = {"import" ~ "\"" ~ import_source ~ "\"" ~ ("as" ~ identifier)? ~ NEWLINE+}
+import_source = @{(!"\"" ~ ANY)*}
+function_definition = {"def" ~ identifier ~ "(" ~ parameter_list ~ ")" ~ "->" ~ "(" ~ type_list ~ ")" ~ ":" ~ NEWLINE* ~ statement* }
 
-parameter_list = {parameter ~ ("," ~ parameter)*}
-parameter = {"private"? ~ type_name ~ identifier}
-type_name = @{"field" ~ ("[" ~ WHITESPACE* ~ expression ~ WHITESPACE* ~ "]")* | "bool" ~ ("[" ~ WHITESPACE* ~ "]")* }
+parameter_list = _{(parameter ~ ("," ~ parameter)*)?}
```

Comment:
We do not need the ParameterList as a concept as we have Rust vectors for that, so silence it 😶